### PR TITLE
Display the currently active restrictions in the library update preference

### LINKED ISSUE #4957: [Feature Request] <Show update restriction>
## Why/User Benefit/User Problem
When you set restrictions to the categories who will be update, the restritions is showed , like this: 
image

However when you set restricions for when the update will happen, like this:
image

The restrictions is not showed:
image

## What/Requirements
This feature is just to show the restricions for when the update will happen, like the frist screenshot

### PR DESCRIPTION
**Pull request by gotson**
Closes #4957

Screenshot 2021-05-25 at 14 59 41

Screenshot 2021-05-25 at 14 59 49

Screenshot 2021-05-25 at 14 59 56

I also pulled the `wifi` and `ac` strings into constants.

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsLibraryController.kt [hunk 1]
```diff
@@ -164,6 +165,33 @@ class SettingsLibraryController : SettingsController() {
...
+                            }
+                        }
+                    val restrictionsText = if (restrictions.isEmpty()) {
+                        context.getString(R.string.none)
+                    } else {
+                        restrictions.joinToString()
+                    }
+
+                    summary = buildSpannedString {
+                        append(context.getString(R.string.pref_library_update_restriction_summary))
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsLibraryController.kt:186]: Maybe could just get rid of this line? Not sure if it's clear enough without it.

Comment (gotson) [reply app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsLibraryController.kt:186]: Up to you really.

Comment (arkon) [reply app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsLibraryController.kt:186]: Let's get rid of it.

**Comment by gotson** [comment]
Screenshot 2021-05-26 at 08 26 30

I also removed the english string from `strings.xml`, i believe Weblate will remove it from all translations too afterwards.

File: app/src/main/java/eu/kanade/tachiyomi/data/preference/PreferencesHelper.kt [hunk 2]
```diff
@@ -227,7 +227,7 @@ class PreferencesHelper(val context: Context) {
 
     fun libraryUpdateInterval() = flowPrefs.getInt(Keys.libraryUpdateInterval, 24)
 
-    fun libraryUpdateRestriction() = prefs.getStringSet(Keys.libraryUpdateRestriction, setOf("wifi"))
+    fun libraryUpdateRestriction() = flowPrefs.getStringSet(Keys.libraryUpdateRestriction, setOf("wifi"))
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/data/preference/PreferencesHelper.kt:230]: Use the constant here?

File: app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsLibraryController.kt [hunk 3]
```diff
@@ -164,6 +165,31 @@ class SettingsLibraryController : SettingsController() {
...
+                                else -> it
+                            }
+                        }
+                    val restrictionsText = if (restrictions.isEmpty()) {
+                        context.getString(R.string.none)
+                    } else {
+                        restrictions.joinToString()
+                    }
+
+                    summary = buildSpannedString {
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsLibraryController.kt:185]: Don't need a spanned string anymore.

File: app/src/main/java/eu/kanade/tachiyomi/data/preference/PreferencesHelper.kt (hunk 2 above)
Comment (gotson) [reply app/src/main/java/eu/kanade/tachiyomi/data/preference/PreferencesHelper.kt]: Good catch!

**Review by arkon — APPROVED** [review]

**Comment by arkon** [comment]
Thanks!

**Pull request merged**